javascript/src_js/index.d.ts

```diff
@@ -7,9 +7,9 @@
  * @format
  */
 
-import type {Yoga} from './wrapAsm';
+import type {Yoga} from './wrapAsm.js';
 
-export * from './generated/YGEnums';
-export * from './wrapAsm';
+export * from './generated/YGEnums.js';
```

Comment:
Is this type resolved correctly, importing from `wrapAsm.js` instead of `wrapAsm.d.ts`?

Reply:
Yes, types are binded to file